src/CommandingOfficers/Patch.java

```diff
@@ -53,32 +54,51 @@ public Commander create(GameScenario.GameRules rules)
   private static final int PILLAGE_ATTACK_BUFF = 25;
 
   private LootAbility myLootAbility = null;
+  double myIncomeRatio = 0.0;
 
   public Patch(GameScenario.GameRules rules)
   {
     super(coInfo, rules);
 
     addCommanderAbility(new PatchAbility(this, PLUNDER_NAME, PLUNDER_COST, PLUNDER_INCOME, PLUNDER_ATTACK_BUFF));
     addCommanderAbility(new PatchAbility(this, PILLAGE_NAME, PILLAGE_COST, PILLAGE_INCOME, PILLAGE_ATTACK_BUFF));
+  }
 
+  @Override
+  public void registerForEvents(GameInstance game)
+  {
+    super.registerForEvents(game);
     // Passive - Loot
     myLootAbility = new LootAbility(this);
```

Comment:
Interesting choice to create this inside `registerForEvents()`, but, I mean, I _guess_.